src/shared/lib/OncoKbUtils.ts

```diff
@@ -80,6 +80,28 @@ const LEVELS = {
     all: ['4', 'R3', '3B', '3A', 'R2', '2B', '2A', 'R1', '1', '0']
 };
 
+export const ALTERATIONS = {
+    mutation: "MUTATION",
+};
+
+/**
+ * Generates <query id, indicator> map.
```

Comment:
are these comments autogenerated?  seems like typescript gives us this.  like, instead of putting return type in comment, why not specify with ts?